test/functional/feature_segwit.py

```diff
@@ -157,8 +157,8 @@ def run_test(self):
         self.skip_mine(self.nodes[2], p2sh_ids[NODE_2][WIT_V1][0], True)  # block 427
 
         self.log.info("Verify unsigned p2sh witness txs without a redeem script are invalid")
-        self.fail_accept(self.nodes[2], "mandatory-script-verify-flag", p2sh_ids[NODE_2][WIT_V0][1], False)
-        self.fail_accept(self.nodes[2], "mandatory-script-verify-flag", p2sh_ids[NODE_2][WIT_V1][1], False)
+        self.fail_accept(self.nodes[2], "script-verify-flag-failed (Operation not valid with the current stack size)", p2sh_ids[NODE_2][WIT_V0][1], False)
+        self.fail_accept(self.nodes[2], "script-verify-flag-failed (Operation not valid with the current stack size)", p2sh_ids[NODE_2][WIT_V1][1], False)
```

Comment:
Same here, since you touch this line anyway

Reply:
added